### PR TITLE
Add different port option.

### PR DESCRIPTION
**Pull request by nirgn975**
#9

### CODE REVIEW

**Comment by nirgn975** [comment]
@RoySegall Ready for code review.

File: solr.sh [hunk 1]
```diff
@@ -16,11 +16,17 @@ do
     echo "options:"
     echo "-b, --background=TRUE / FALSE    specify if solr will execute in the background or not."
     echo "-s, --source=The Source          specify the source destination to download solr from."
+    echo "-p, --port=The port number       specify the port number for the solr script before the -b flag (if you use it)."
     exit 0
   fi;
 
+  if [[ $var == "-p" || $var == "--port" ]]; then
+    solr_command+=" -Djetty.port="${!i}
+    echo $solr_command
```
Comment (RoySegall) [inline solr.sh]: ?

**Comment by nirgn975** [comment]
@RoySegall Ready.

File: solr.sh (hunk 1 above)
Comment (nirgn975) [inline solr.sh]: It was like that when I got there (;

Comment (RoySegall) [inline solr.sh]: Yeah... the plus sign framed you

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Write something about your project, roy.
 ## Options
 1. `-b | --background` - Add the flag to activate solr in the background. Example: `./solr.sh -b`.
 2. `-s | --source` - Add the flag to download solr from another destination and specify it. Example: `./solr.sh -s http://someurl.com/solr-4.7.2`.
-
+3. `-p | --port` - Add the flag activate the solr script in different port number (if you use `-b` flag, use this flag first). Example: `./solr.sh -p 4000`.
```
Comment (RoySegall) [inline README.md]: Add example for backround

**Comment by nirgn975** [comment]
@RoySegall Done. Ready for review.

**Pull request closed without merging**